blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineNodeGraphVisitor.java

```diff
@@ -53,8 +53,13 @@
 
 /**
  * @author Vivek Pandey
+ * <p>
+ * Run your Jenkins instance with <code>-DNODE-DUMP-ENABLED</code> to turn on the logging when diagnosing bugs! You'll
+ * also need to have a logging config that enables debug for (at least) this class. Use
+ * <code>-Djava.util.logging.config.file=./logging.properties</code> to set a custom logging properties file from the
+ * command line, or do it from within the admin UI.
```

Comment:
might fail javadoc as <p> it's not closed

Reply:
Done :)